server/src/modules/admin/admin.controller.ts

```diff
@@ -119,8 +119,8 @@ export class AdminController {
     @ApiOperation({ description: 'Retrieve a list of maps' })
     @ApiOkPaginatedResponse(MapDto, { description: 'Paginated list of maps' })
     @ApiBadRequestResponse({ description: 'Invalid query data' })
-    getMaps(@Query() _query: MapsCtlGetAllQuery): Promise<PaginatedResponseDto<MapDto>> {
-        return void 0;
+    getMaps(@Query() _query: AdminCtlMapsGetAllQuery): Promise<PaginatedResponseDto<MapDto>> {
+        return this.adminService.getMaps(_query.priority, _query.status, _query.skip, _query.take, _query.expand);
```

Comment:
Underscore prefixes in var names here were just to stop ESLint complaining, now they're actually being used they should be removed.